resolwe_bio/processes/expression/cuffnorm.yml

```diff
@@ -104,20 +104,23 @@
       experiment='{% for experiment in cuffquant %}{{ experiment.cxb.file }},{% endfor %}'
       replicates='{% for r in replicates %}{{r}},{% endfor %}'
       labels='{% for l in labels %}{{l}},{% endfor %}'
+      sample_names='{% for sample in cuffquant %}{{sample|sample_name}},{% endfor %}'
 
       labels=${labels%?}
```

Comment:
Are lables still needed? Can you put labels out of processor and use a string of sample names for labels?